source/contracts/reporting/ReputationToken.sol

```diff
@@ -31,7 +31,7 @@ contract ReputationToken is DelegationTarget, ITyped, Initializable, VariableSup
     }
 
     // AUDIT: check for reentrancy issues here, _destination will be called as contracts during validation
-    function migrateOut(IReputationToken _destination, address _reporter, uint256 _attotokens) public afterInitialized returns (bool) {
+    function migrateOut(IReputationToken _destination, address _reporter, uint256 _attotokens) public onlyInGoodTimes afterInitialized returns (bool) {
```

Comment:
I think everything except the ERC20 token part of this contract (and VariableSupply mint/burn since those are needed for the escape) should be `onlyInGoodTimes` (meaning all `public`/`external` non `view`/`pure` methods).  The same goes for the other tokens as well.